test/unit/generator/content/part2/map-reductions.test.js

```diff
@@ -0,0 +1,116 @@
+const mapReductions = require('../../../../../app/generator/content/part2/map-reductions')
+
+describe('map reductions', () => {
+  test('returns empty array if no reductions', () => {
+    const result = mapReductions([{}])
+    expect(result.length).toBe(0)
+  })
+
+  test('returns empty array if empty array of reductions', () => {
+    const result = mapReductions([{ reductions: [] }])
+    expect(result.length).toBe(0)
+  })
+
+  test('returns empty array if no funding', () => {
+    const result = mapReductions([])
+    expect(result.length).toBe(0)
+  })
+
+  test('returns reductions if one reduction', () => {
+    const result = mapReductions([{ reductions: [{ reason: 'Late claim submission' }] }])
```

Comment:
Thoughts on using `mockReductions` here and throughout?

Maybe alludes to best having the reasons as `constants` in `app` and importing them into `mockReductions` and here, whatcha think?

Reply:
The mock reductions object is mocking the output of the function this test class is testing.  So couldn't use it here.

I did originally use the `mockStatement` object for a few of the tests, but without adding more reductions to it to cover all of these test scenarios all it brought was inconsistency and difficulty understanding what the input to the test was.